### DISCUSSION TITLE
I bought Github PRO by mistake and I can't cancel and get my money back, can anyone help me?

### DISCUSSION THREAD

**Discussion by devsantosandre** [community/community#32220] [Other Feature Feedback, Questions, & Ideas] [answered]
9 days ago I try to contact Github support and I have no success, I accidentally subscribed to the annual PRO plan and I can't cancel and get my money back

**Comment by melvincwng** [comment, marked as answer]
@devsantosandre Hi Andre, please follow the steps in this [GitHub Documentation](https://docs.github.com/en/billing/managing-billing-for-your-github-account/downgrading-your-github-subscription) link to cancel/downgrade your GitHub Subscription plan.

Hope this clarifies & answers your question 😃 

Have a good day ahead!

**Reply by nazmuhammadansarnaz91-art** [reply]
Support sheerid of the sitting update on the GitHub Docs in supply in

**Reply by tommyllama** [reply]
I also accidentally upgrade my personal account to pro while I already have team account/license in my org. I downgraded immediately, but see the bill now. How can I get refund?

**Reply by gcasillas** [reply]
After cancelingGithub pro,  how long will it remain in a pending status?

**Reply by Sunil56224972** [reply]
> I also accidentally upgrade my personal account to pro while I already have team account/license in my org. I downgraded immediately, but see the bill now. How can I get refund?

step 1: Go to [support.github.com/contact](https://support.github.com/contact)

Step 2: Select "Billing & Payments"

**Step 3: Write something like:

Hi, I accidentally purchased GitHub Pro. I didn't mean to buy it and haven't really used any Pro features. Can you please cancel it and refund my money? My username is [your-username]. Thanks!"*

Step 4: Hit send and wait — they usually reply in 1-2 business days and give the refund.

That's it bro. Just message their support, be chill, tell them it was a mistake, and they'll sort it out. 👍

**Comment by harundemirkaya** [comment]
https://github.com/pricing
If you contact the address, they will help you on the subject.

**Comment by Lilianada** [comment]
Hi I also mistakenly for this
GITHUB RECEIPT - ORGANIZATION PURCHASE 

1 additional seat ($48/year each - prorated for 366 days)

Total: $44.00 USD*

I want to cancel it now and get my money back in full, how do I go about this?

**Comment by 4tunexx** [comment]
I purchase by accident too! now cant contact support and cant go back to free user keeps saying on downgrading
The plan change was successful. @4tunexx will be updated to the pro yearly plan.
I want to downgrade to free! AND refund! i wanted Copilot Pro not github pro its so confusing!

**Reply by engoncode** [reply]
same problem with me broh

**Reply by Sunil56224972** [reply]
step 1: Go to [support.github.com/contact](https://support.github.com/contact)

Step 2:  Select **"Billing & Payments"**

**Step 3: Write something like:

Hi, I accidentally purchased GitHub Pro. I didn't mean to buy it and haven't really used any Pro features. Can you please cancel it and refund my money? My username is [your-username]. Thanks!"*

Step 4: Hit send and wait — they usually reply in **1-2 business days** and give the refund.



That's it bro. Just message their support, be chill, tell them it was a mistake, and they'll sort it out. 👍

**Comment by Ciantic** [comment]
Funnily enough, I also wanted to buy CoPilot Pro and accidentally bought GitHub Pro.

The offering between Copilot Pro and Github Pro are a bit confusing depending on what pages one navigates. Because I was reading about Copilot on Github, but somehow navigated to Github Pro purchase...

**Edit** I got refund through ticket system, note that ticket system has one confusing part as well. It doesn't create the ticket until you find the small button at the bottom after the AI answer it throws. I first thought I created ticket, but didn't actually make it.

**Reply by PineVNM** [reply]
Hello, can you show me how to raise the ticket cause I have the same mistaken purchase?

**Reply by Code-Milan** [reply]
I agree. I made the same mistake.

**Comment by LilMaia** [comment]
Me too, i opened a ticket, almost one month, and no answers.

**Comment by xxxtumxxx** [comment]
The same problem.  This is due to confusion between GitHub Pro and GitHub Copilot.
How do I avoid being charged Pro plan annually?  Now i downgrade to free but still display annually billing.

**Comment by 4tunexx** [comment]
Just wait for refund it will come support is busy I got my refund thanks GitHub

**Comment by tearerry** [comment]
Really I got the same problem I got confused between GitHub Pro and GitHub Copilot too.
I have pressed downgrade to free but still display annually billing and no refund...

**Comment by mesutdinler** [comment]
> Just wait for refund it will come support is busy I got my refund thanks GitHub 

okay, i'll wait

**Comment by LokiCastiel** [comment]
How long would they response? It's been 5 days since I've created the ticket and still got no response.

**Reply by mesutdinler** [reply]
"It's been three days for me. Unfortunately, there's nothing I can do but wait."
